modPrecompiled/src/org/aion/precompiled/ContractFactory.java

```diff
@@ -28,6 +28,7 @@
 import org.aion.mcf.core.AccountState;
 import org.aion.mcf.db.IBlockStoreBase;
 import org.aion.precompiled.contracts.ATB.TokenBridgeContract;
+import org.aion.precompiled.contracts.EDVerifyContract;
```

Comment:
Ahmmed Aion Chowdhury 
From Bangladesh 
Aion Not only the name 
Aion is world famous Brand for Crypto world (Cryptocurrency)